src/pipecat/services/gladia.py

```diff
@@ -135,13 +57,92 @@ def __init__(
         self._url = url
         self._settings = {
             "sample_rate": params.sample_rate,
-            "language": params.language if params.language else Language.EN,
+            "language": self.language_to_service_language(params.language)
+            if params.language
+            else Language.EN,
```

Comment:
this indentation looks weird. very well could be correct but wanted to call it out 🤔

Reply:
This is what the linter wants 🤷